sass/includes/_transcription.scss

```diff
@@ -0,0 +1,147 @@
+.transcription {
+  background: $color__grey-2;
+  color: $color__white;
+  padding-bottom: 0.5rem;
+
+  .hidden {
+    display: none;
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 4rem;
+    padding-top: 4rem;
+    margin-right: auto;
+    margin-left: auto;
+
+    max-width: 80%;
+
+    @media screen and (min-width: map-get($grid-breakpoints, "lg")) {
+      display: grid;
+      row-gap: 1.875rem;
+      column-gap: 3rem;
+      grid-template-columns: 55% 45%;
+    }
+  }
+
+  &__text {
+    margin-top: 0;
+
+    @media screen and (min-width: map-get($grid-breakpoints, "lg")) {
+      h2 {
+        margin-top: 0;
+      }
+    }
+  }
+
+  &__preview {
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 4rem;
+    display: block;
+    position: relative;
+  }
+
+  &__underline {
+    width: 96%;
+    display: block;
+    position: absolute;
+    bottom: 0;
+
+    @media screen and (min-width: "lg") {
+      max-width: 100%;
+    }
+  }
+
+  &__image-counter {
+    text-align: center;
+    margin-top: 1.5em;
+  }
+
+  &__sensitive_image {
+    filter: blur(8px);
+    -webkit-filter: blur(8px);
+  }
+
+  &__open {
+    @extend .tna-button--yellow;
+    position: absolute;
+    left: 50%;
+    -webkit-transform: translateX(-50%);
+    transform: translateX(-50%);
+    bottom: -5%
+  }
+
+  &__close {
+    display: block;
+    padding: 1em;
+    background-color: inherit;
+    border: none;
+    color: $yellow;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 2rem;
+
+  }
+
+  &__icon {
+    fill: $yellow;
+    width: 2rem;
+    height: 2rem;
+    padding-left: 0.3rem;
+  }
+
+  &__image {
+    max-width: 96%;
+    height: auto;
+
+    @media screen and (min-width: "lg") {
+      max-width: 100%;
+    }
+  }
+
+  &__tablist {
+    max-width: 65em;
+    font-family: "lucida grande", sans-serif
+  }
+
+  &__tab {
+    background-color: inherit;
+    border: none;
+    display: inline-block;
+    margin: 0.125rem;
+    color: $color__white;
+    font-size: $h2-font-size;
+    font-weight: $headings-font-weight;
+
+    @media screen and (min-width: map-get($grid-breakpoints, "lg")){
+      font-size: 1.4rem;
+    }
+
+    @media screen and (min-width: map-get($grid-breakpoints, "xl")){
+      font-size: 1.8rem;
+    }
+
+    &:first-of-type {
+      margin-right: 1.875rem;
+    }
+
+    &:focus,
+    &:hover,
+    &[aria-selected="true"] {
+      border-bottom: $color__white solid 3px;
+    }
+
+  }
+
+  &__tabpanel {
+    margin-top: 2.5rem;
+    color: $color__white;
+
+    >p {
```

Comment:
I'd have to nest this class > rich-text > p to get this without this selector. Feels cleaner, what do you think @janehughes ?

Reply:
You're right. It's been turned off in the latest version so it now works. Yep good spot, added that.